db/migrate/20230606061826_create_kit_allocations.rb

```diff
@@ -0,0 +1,11 @@
+class CreateKitAllocations < ActiveRecord::Migration[7.0]
+  def change
+    create_table :kit_allocations do |t|
+      t.references :organization, null: false, foreign_key: true
+      t.references :storage_location, null: false, foreign_key: true
+      t.references :kit, null: false, foreign_key: true
+      t.integer :kit_allocation_type, null: false, default: 0
```

Comment:
Is there a reason we can't use Postgres enums here?

Reply:
@dorner updated the code with Postgres enum